jatszohaz/jatszohaz/views.py

```diff
@@ -65,6 +67,9 @@ class ProfileView(LoginRequiredMixin, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        if self.request.user.has_perm('jatszohaz.view_all'):
+            context['comment_form'] = NewCommentForm()
+            context['comments'] = UserComment.objects.all().order_by('created').filter(user=self.get_object())

```

Comment:
Azért gondoltam, hogy a legrégebbi legyen elöl mert a kölcsönzéses kommenteknél is úgy van, de nekem mindegy a sorrend.

Reply:
Oksa, jogos.